asreview/webapp/src/Components/ExportDialog.js

```diff
@@ -119,12 +119,18 @@ const ExportDialog = (props) => {
             value={exportFileType}
             onChange={handleExportFileTypeChange}
           >
-            <MenuItem value={"ris"}>RIS (UTF-8)</MenuItem>
+            <MenuItem value={"xlsx"}>Excel</MenuItem>
             <MenuItem value={"csv"}>CSV (UTF-8)</MenuItem>
             <MenuItem value={"tsv"}>TSV (UTF-8)</MenuItem>
-            <MenuItem value={"excel"}>Excel</MenuItem>
+            <MenuItem value={"ris"}>RIS (UTF-8)</MenuItem>
           </Select>
         </Box>
+        {exportFileType === "ris" && (
+          <Alert severity="info">
+            Experimental feature: available only if you imported the RIS file
+            when creating the project
```

Comment:
I think it should be: ... if you imported **a** RIS file ...

Reply:
Yeah, done!